packages/input-option/src/InputOption/InputOption.style.ts

```diff
@@ -0,0 +1,132 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  fontFamilies,
+  spacing,
+  transitionDuration,
+  typeScales,
+} from '@leafygreen-ui/tokens';
+
+export const inputOptionStyles = css`
+  position: relative;
+  list-style: none;
+  outline: none;
+  border: unset;
+  margin: 0;
+  text-align: left;
+  text-decoration: none;
+
+  font-size: ${typeScales.body1.fontSize}px;
+  line-height: ${typeScales.body1.lineHeight}px;
+  font-family: ${fontFamilies.default};
+  padding: ${spacing[2]}px ${spacing[2] + spacing[1]}px;
```

Comment:
`spacing[2] + spacing[1]` feels kind of awkward. I'm wondering if we should evaluate having our spacing units just jump in increments of 4px if that's our base.

Reply:
Definitely agree that `12px` should be in our spacing tokens, since it's used so often

Reply:
Filed a ticket to address this https://jira.mongodb.org/browse/LG-2857